completions/asdf.bash

```diff
@@ -54,9 +54,10 @@ _asdf() {
     COMPREPLY=($(compgen -W "--head" -- "$cur"))
     ;;
   uninstall | where | reshim)
-    if [[ "$plugins" == *"$prev"* ]]; then
+    if [[ " $plugins " == *" $prev "* ]]; then
       local versions
-      versions=$(asdf list "$prev" 2>/dev/null)
+      # The first two columns are either blank or contain the "current" marker.
+      versions=$(asdf list "$prev" 2>/dev/null | colrm 1 2)
```

Comment:
Is `colrm` available on most systems? Curious if it's available on all Linux distros and MacOS.

Reply:
It's available on my MacOS machine (Sonoma 14.5) and my Ubuntu madchine (Linux 5.4.0).

Reply:
Good catch, I hadn't even thought about that. 😅

Fortunately, it looks to be pretty universally available. It's part of util-linux, so should be part of every Linux distro since forever. Both the Linux and FreeBSD man pages identify `colrm` as having been part of ”3.0 BSD", which I think predates the Net/Free/Open split. That would put it in every version of OS X / macOS, as well. Oracle's man page even describes it as being part of Solaris (via util-linux again).

It's not clear to me whether the Linux and BSD variants are separate code bases (util-linux comes from <kernel.org> and the BSD man page indicates `colrm` is GNU software), but I'm not seeing anything about compatibility issues. `colrm` is a pretty ancient tool and very simple as well, after all.

Reply:
Thanks, I've not used `colrm` before but it seems like a good tool for situations like this. I'm going to merge this now, I'm sure if it breaks someone will complain :laughing: .